contracts/chain-adapters/Arbitrum_Adapter.sol

```diff
@@ -166,12 +168,18 @@ contract Arbitrum_Adapter is AdapterInterface {
      * @param _l1ArbitrumInbox Inbox helper contract to send messages to Arbitrum.
      * @param _l1ERC20GatewayRouter ERC20 gateway router contract to send tokens to Arbitrum.
      * @param _l2RefundL2Address L2 address to receive gas refunds on after a message is relayed.
+     * @param _l1Usdc USDC address on L1.
+     * @param _circleDomain Circle domain set for this chain. NOTE: this is issued by circle and is irrelevant of chain id
```

Comment:
What is an example circle domain? Can/should we make this a constant in the CCTPAdapter?

Reply:
It's a chain identifier (agnostic of main/testnet) that is issued by circle. I have docs here that explain/link to the official docs. I'm not sure we should hardcode it because it would add a lookup, but it's essentially circle's version of a chain Id

https://github.com/across-protocol/contracts-v2/blob/7b2a113a5789875972eda7b7f2ed9e2cdfa73500/deploy/consts.ts#L124-L130

Reply:
what do you mean it would add a lookup? Couldn't we hardcode the chain ID's that we pass to the CCTP_Adapter in each child adapter?

Reply:
Resolved offline - this is a fair point. Done: https://github.com/across-protocol/contracts-v2/pull/391/commits/83c2d22af9e9fdf0faf6b632b4b6f9e1142464be